pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -1507,5 +1507,39 @@ self: super: {
   # 2020-06-24: Tests are broken in hackage distribution.
   # See: https://github.com/robstewart57/rdf4h/issues/39
   rdf4h = dontCheck super.rdf4h;
+  
+  # hasn't bumped upper bounds
+  # test fails: "floskell-test: styles/base.md: openBinaryFile: does not exist (No such file or directory)"
+  floskell = dontCheck (doJailbreak super.floskell);
+
+  # hasn't bumped upper bounds
+  # test fails because of a "Warning: Unused LANGUAGE pragma"
+  monad-dijkstra = dontCheck (doJailbreak super.monad-dijkstra);
+
+  # only used by haskell-language-server as of now
+  # Tests disabled: there is an issue with Cabal 3
+  #   -> it builds in `./dist` while it needs `./dist-newstyle` for the tests
+  lsp-test_0_11_0_2 = dontCheck super.lsp-test_0_11_0_2;
+
+  # haskell-language-server uses its own fork
+  hls-ghcide = 
+    dontCheck (
+      (self.callPackage (import ../tools/haskell/haskell-language-server/hls-ghcide.nix) { }).override {
+        # specific versions required
+        lsp-test = self.lsp-test_0_11_0_2;
+      }
+    );
```

Comment:
If appropriate, could you report these failed tests (and upper bound problems) upstream, and then link inline to the upstream issues?